src/components/UserProfile.js

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState } from "react";
+import { useParams, useHistory } from "react-router-dom";
+import GhPolyglot from "gh-polyglot";
+import dotenv from "dotenv";
+
+import UserDetails from "./UserDetails";
+import Loader from "./Loader";
+import RepoList from "./RepoList";
+import UserGraphs from "./UserGraphs";
+import Footer from "./Footer";
+
+dotenv.config();
```

Comment:
I think this should go in the App.js? Or in a util/configuration file somewhere.